changelog/185.breaking.rst

```diff
@@ -0,0 +1,2 @@
+Asda and Lamb_Oseen classes were both declassed.
+Now instead of initiating the classes the methods that were previously provided within the class can be used directly with the same exact functionality.
```

Comment:
```suggestion
The ``Asda`` and ``Lamb_Oseen`` classes have been removed and replaced with functions that replicate the original process.
```

Reply:
There should be some sort of transition guide, to how a user how to change their code if they are using it.

Reply:
ok, should I put it in the changelog?

Reply:
Yeah, I think so